src/all.js

```diff
@@ -4,13 +4,29 @@ import TimeoutDialog from './components/timeout-dialog/timeout-dialog';
 import UserResearchBanner from './components/user-research-banner/user-research-banner';
 import { nodeListForEach } from './common';
 import SessionActivityService from './components/timeout-dialog/session-activity-service';
+import HmrcPrintLink from './components/hmrc-print-link/hmrc-print-link';
 
 function initAll() {
   const $AccountMenuSelector = '[data-module="hmrc-account-menu"]';
   if (document.querySelector($AccountMenuSelector)) {
     new AccountMenu($AccountMenuSelector).init();
   }
 
+  const $HmrcPrintLinkArefSelector = 'a[data-module="hmrc-print-link"]';
+  if (document.querySelector($HmrcPrintLinkArefSelector)) {
+    new HmrcPrintLink($HmrcPrintLinkArefSelector, window).init();
+  }
+
+  const $HmrcPrintLinkButtonSelector = 'button[data-module="hmrc-print-link"]';
+  if (document.querySelector($HmrcPrintLinkButtonSelector)) {
+    new HmrcPrintLink($HmrcPrintLinkButtonSelector, window).init();
+  }
+
+  const $HmrcPrintLinkInputButtonSelector = 'input[type="button"][data-module="hmrc-print-link"]';
+  if (document.querySelector($HmrcPrintLinkInputButtonSelector)) {
+    new HmrcPrintLink($HmrcPrintLinkInputButtonSelector, window).init();
+  }
+
```

Comment:
it's unlikely that people will use more than one of these on a page, but maybe we could loop through these instead like we do for back links. You can also provide a comma separated list of selectors to querySelectorAll so rather than these three. The convention of the other ones too is that rather than the module doing the initial query selecting, you do it here, so `const printLinks = document.querySelectorAll("...")`
```
  const $BackLinks = document.querySelectorAll('[data-module="hmrc-back-link"]');
  nodeListForEach($BackLinks, ($BackLink) => {
    new BackLinkHelper($BackLink, window, document).init();
  });
```